pkgs/top-level/aliases.nix

```diff
@@ -151,6 +151,7 @@ mapAliases ({
   composable_kernel = throw "'composable_kernel' has been replaced with 'rocmPackages.composable_kernel'"; # Added 2023-10-08
   cpp-ipfs-api = cpp-ipfs-http-client; # Project has been renamed. Added 2022-05-15
   crispyDoom = crispy-doom; # Added 2023-05-01
+  cryptowatch-desktop = throw "Cryptowatch Desktop was sunset on September 30th 2023 and has been removed from nixpkgs";
```

Comment:
I don't think there are guidelines about it but feels like we should add  `# Added 2023-12-22`

Reply:
I agree, fixed in https://github.com/NixOS/nixpkgs/pull/275945/commits/f00208148de5ea9440e0bcf8a6cf852cbe88f635